polaris-react/src/components/Collapsible/tests/Collapsible.test.tsx

```diff
@@ -132,6 +132,40 @@ describe('<Collapsible />', () => {
       ).not.toBeNull();
     });
   });
+
+  describe('preventMeasuringOnChildUpdate', () => {
+    it('sets maxHeight to `none` and overflow to `visible` preventMeasuringOnChildUpdate is true', () => {
+      const collapsible = mountWithApp(
+        <CollapsibleWithPreventMeasuringOnChildUpdate
```

Comment:
I think we can leverage `setProps()` to update the children and re-phrase the test title around measuring/no-measuring

```
    it('does not re-measure on child update when preventMeasuringOnChildUpdate is true', () => {
      const id = 'test-collapsible';

      const collapsible = mountWithApp(
        <Collapsible id={id} open preventMeasuringOnChildUpdate>
          <div>Foo</div>
        </Collapsible>,
      );

      collapsible.setProps({children: <div>Bar</div>});

      expect(collapsible).toContainReactComponent('div', {
       id,
        style: {maxHeight: 'none', overflow: 'visible'},
      });
    });

    it('re-measures on child update when preventMeasuringOnChildUpdate is false', () => {
      const id = 'test-collapsible';

      const collapsible = mountWithApp(
        <Collapsible id={id} open preventMeasuringOnChildUpdate={false}>
          <div>Foo</div>
        </Collapsible>,
      );

      collapsible.setProps({children: <div>Bar</div>});

      expect(collapsible).toContainReactComponent('div', {
       id,
        style: {maxHeight: '0px', overflow: 'hidden'},
      });
    });
```